app/views/admin/events/index.html.erb

```diff
@@ -18,19 +18,18 @@
 
   <tbody>
     <% @events.each do |event| %>
+    <% locationType = JSON.parse(event.location) %>
```

Comment:
This will break if "event.location" returns nil, which is the case in my database for my legacy records. So a default is wise. In ruby there are a few ways to do this (also default variable names in ruby is to use snake_case, instead of camelCase):

`location_type = event.location ? JSON.parse(event.location) : nil`
or
`location_type = JSON.parse(event.location || {})`

Either way I just noticed that this might break bellow.... check next comment.